File: core/src/com/dog/game/Stats.java
package com.dog.game;

/**
 * Created by spkerkela on 07/05/2017.
 */
public class Stats {
    public int hitPoints;
    public int mana;
    public int strength;

    @Override
    public String toString() {
        return String.format("HP: %s, MANA: %s, STR: %s", hitPoints, mana, strength);
    }
}

File: core/src/com/dog/game/Stats.kt
package com.dog.game

/**
 * Created by spkerkela on 07/05/2017.
 */
class Stats {
    var hitPoints: Int = 0
    var mana: Int = 0
    var strength: Int = 0

    override fun toString(): String {
        return String.format("HP: %s, MANA: %s, STR: %s", hitPoints, mana, strength)
    }
}
